gobblin-modules/gobblin-couchbase/scripts/install_test_deps.sh

```diff
@@ -10,5 +10,9 @@ if [ -d mock-couchbase/.git ];
     git -C mock-couchbase checkout b929a9f99a31a233c43bd94b6a696ad966206e02
   fi
 pushd mock-couchbase
-mvn package -Dmaven.test.skip=true
```

Comment:
Should we explicitly check whether mvn is installed? That would allow us to provide a meaningful error message below. [mvn missing versus mvn build failed]